plugins/reports/views/locale/el_GR/locale.po

```diff
@@ -1,8 +0,0 @@
-# Manolis Saldaris <omanos@gmail.com>, 2022.
```

Comment:
Looks like the `plugins/reports/views/` has been removed, just this locale remained.